web-animations-2/Overview.bs

```diff
@@ -144,9 +144,48 @@ Along with the following updated description:
 </div>
 
 
+<h3 id="timelines">Timelines</h3>
+
+Add:
+
+> The <dfn lt="timeline duration">duration</dfn> of a timeline gives the
+> maximum value a timeline may generate for its current time. This value is
+> used to calculate the [=intrinsic iteration duration=] for the target effect
+> of an animation that is associated with the timeline when the effect's
+> [=iteration duration=] is 'auto'. The value is computed such that the effect
+> fills the available time.  For a monotonic timeline, there is no upper bound
+> on current time, and [=timeline duration=] is unresolved. For a progress based
+> timeline, the [=timeline duration=] is 100%.
+
 <h3 id="animations">Animations</h3>
 
-<div class="informative-bg"><em>This section is non-normative</em>
+Add:
+
+> Animation effects associated with a progress-based timeline require their
+> timing properties to be converted to proportions. The procedure for converting
+> a <dfn lt="time-based animation to a proportional animation"></dnf> is as
+> follows:
+>
+> 1.  If the [=iteration duration=] is auto, then perform the following steps.
+>
+>     1.   Compute [=start delay=] and [=end delay=] to 0, as it is not possible
+>          to mix time and proportions. Future versions may allow these properties
+>          to be assigned percentages.
```

Comment:
We would need to rerun the conversion process anytime timing properties are updated. This could be through a CSS style change or call to updateTiming on the AnimationEffect. I can call this out in a note (issue?) and then followup with a separate PR to flush out each place we need to renormalize the timing.

My intent in the future was not to allow mixing of times and percentages, but instead to allow percentage based start and end delays (these are currently double and not CSSNumberish).  We would be able to have all timings as percentages or in time units.  At present, the only ways we would get mixed units are with group/sequence effects or duration = auto.

The proposal here is to ignore time based start and end delays if duration is percentage based. These would continue to be ignored in the future as only percentage based start and end delays would be compatible with a percentage based duration.

Reply:
Added a step for updateTiming to call the normalization procedure as well as a note that normalization is required after tweaking a timing parameter via a CSS animation property.